ui-v2/src/utils/states.ts

```diff
@@ -0,0 +1,18 @@
+import type { components } from "@/api/prefect";
+
+export const RUN_STATES = {
+	COMPLETED: "Completed",
+	RUNNING: "Running",
+	SCHEDULED: "Scheduled",
+	PENDING: "Pending",
+	FAILED: "Failed",
+	CANCELLED: "Cancelled",
+	CANCELLING: "Cancelling",
+	CRASHED: "Crashed",
+	PAUSED: "Paused",
+} as const satisfies Record<
+	components["schemas"]["StateType"],
+	Capitalize<Lowercase<components["schemas"]["StateType"]>>
+>;
```

Comment:
upon reviewing my own PR, i may want to remove this, but I couldn't find a type like this that existed already

Reply:
ok i think `ICONS` maps state types to icon names where i am wanting state types to display names, so I think these are distinct

Reply:
This file looks like it has a similar map: https://github.com/PrefectHQ/prefect/blob/main/ui-v2/src/components/flow-runs/flow-runs-list/flow-runs-filters/state-filters.constants.ts. I think it'd be worthwhile to have some form of this in `src/api/flow-runs` for broader consumption.

Reply:
i made a `src/api/flow-runs/constants.ts` for my StateType -> display name map, since the flow runs filters map seemed a bit different and mapped the opposite way - lmk what you think!